Sources/WebAuthn/Ceremonies/Registration/PublicKeyCredentialCreationOptions.swift

```diff
@@ -38,11 +62,9 @@ public struct PublicKeyCredentialCreationOptions: Encodable, Sendable {
     /// preferred.
     public let publicKeyCredentialParameters: [PublicKeyCredentialParameters]
 
-    /// A time, in seconds, that the caller is willing to wait for the call to complete. This is treated as a
+    /// A time, in milliseconds, that the caller is willing to wait for the call to complete. This is treated as a
     /// hint, and may be overridden by the client.
-    ///
-    /// - Note: When encoded, this value is represented in milleseconds as a ``UInt32``.
-    public let timeout: Duration?
+    public let timeoutInMilliseconds: Int64?
```

Comment:
Note that this was intentional so we expose a more Swift-native API for timing, and the struct encodes with milliseconds for easy integration with JS.

Reply:
Got it, makes sense. I'll adjust that one.

Any thoughts on how to introduce this change, given that it might change behaviour for those who are relying on the current implementation? My thoughts:

* A version change, I think this is quite a critical part of the lib, so might be the proper course. This would require immediate action from clients when they update the package.
* I could come up with some layer of abstraction (maybe a protocol or a class-cluster/factory like approach) that provides current behaviour by default, and the extended behaviour when some option is specified or a specific initialiser is called.  This would require zero effort from clients, but complicates the code base a little bit and keeps essentially an incompletely implemented spec alive.
* A combination of both above, where the new default is the extended version, but current version can still be requested as well. This would require little, but non-zero effort of clients using the current version when updating the library. This could be mitigated by changing some function signature (`beginRegistration ` comes to mind) to take an option so that when the library gets updated, clients will _have_ to specify which behaviour the want

Reply:
The package is in prerelease, so breaking changes can still be made (and probably should if they improve the API). That said, not sure what about these changes (unless there are more) would incur breaking changes — it mostly seems like a collection of added properties more than anything, all of which could have sensible defaults?

Reply:
Yes, that is what I am thinking as well. 
However: maybe, some implementations were relaying for example on the effect on android that I found undesirable: users being able to authenticate once, but unable to re-use that credential on subsequent logins.